src/api.coffee

```diff
@@ -49,12 +49,12 @@ module.exports = (application) ->
 
 	unparsedRouter.post '/v1/reboot', (req, res) ->
 		utils.mixpanelTrack('Reboot')
-		request.post(config.gosuperAddress + '/v1/reboot')
+		utils.gosuper.post('/v1/reboot')
 		.pipe(res)
 
 	unparsedRouter.post '/v1/shutdown', (req, res) ->
 		utils.mixpanelTrack('Shutdown')
-		request.post(config.gosuperAddress + '/v1/shutdown')
+		utils.gosuper.post('/v1/shutdown')
 		.pipe(res)
```

Comment:
This should have an error handler, otherwise the supervisor could crash from a stream piping error.